web/satellite/tests/unit/common/SearchComponent.spec.ts

```diff
@@ -48,4 +48,20 @@ describe('SearchComponent.vue', () => {
         expect(onMouseLeaveSpy.callCount).toBe(1);
         expect(processSearchQuerySpy.callCount).toBe(1);
     });
+
+    it('function clearSearch works correctly', () => {
+        let processSearchQuerySpy = sinon.spy();
+
+        const wrapper = mount(SearchComponent, {
+            methods: {
+                processSearchQuery: processSearchQuerySpy,
+            }
+        });
+
+        wrapper.vm.clearSearch();
+
+        expect(wrapper.vm.$data.searchQuery).toMatch('');
```

Comment:
is `searchQuery` something other than `''` before this test case?On line 31 it looks like you are also checking the same thing, and I don't see what has changed since then.

Reply:
Agreed